Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.cpp

```diff
@@ -441,25 +439,17 @@ int CLuaCameraDefs::GetCameraClip(lua_State* luaVM)
     return 2;
 }
 
-int CLuaCameraDefs::SetCameraViewMode(lua_State* luaVM)
+bool CLuaCameraDefs::SetCameraViewMode(int usVehicleViewMode, std::optional<int> usPedViewMode)
```

Comment:
same here, it should be `unsigned `short as well, not int, since you've used it before

Reply:
I think should be `unsigned char` there but new lua parser did not work with this type. I spend a lot of time here. I will change in `unsigned chrar` if it will works.

Reply:
ohh, yeah. chars aren't supported. You could use unsigned short. as `us` stands for taht. Btw, tbh, I'd just leave out the hungarian notation, but since you've already included it, just fix it.

Reply:
I will do this after merging #1555

Reply:
If that ever gets merged...